fecgov/fec-cms
# AF canonical page updates

## Summary

- Resolves #4552 
- Resolves #4650 

Incorporates requested changes for the admin fine canonical pages

## Screenshots

Canonical page with no data for the disposition table

![Screen Shot 2021-07-12 at 11 29 54 AM](https://user-images.githubusercontent.com/12799132/125314782-807d4600-e304-11eb-8f45-8368eb2881eb.png)



Canonical page with data for the disposition table

![Screen Shot 2021-07-12 at 11 29 24 AM](https://user-images.githubusercontent.com/12799132/125314721-70fdfd00-e304-11eb-9473-48b292221a0a.png)



### Required reviewers

1 UX, 1 front end, 1 backend

## Impacted areas of the application

General components of the application that this PR will affect:

-  Admin fines canonical page

## How to test

- Checkout this branch
- Set your `FEC_API_URL` to the stage environment
- `npm run build` just in case
- `./manage.py runserver`
- Go to an AF canonical page. Ex: http://localhost:8000/data/legal/administrative-fine/26/, http://localhost:8000/data/legal/administrative-fine/3926/
   - Each section should appear: Summary, Disposition, Documents
   - Header should contain the AF number, committee name, and ID
   - Document links should go to the appropriate file on the S3 bucket. Note that the doc links only work for production, since it's a relative path in the code, so this relative path `/files/legal/admin_fines/26/21092501276.pdf` is this in prod: https://www.fec.gov/files/legal/admin_fines/26/21092501276.pdf
